src/applications/lgy/coe/containers/App.jsx

```diff
@@ -1,12 +1,40 @@
 import React from 'react';
-
+import { connect } from 'react-redux';
+import { isLoggedIn } from 'platform/user/selectors';
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
-export default function App({ location, children }) {
+const FORM_PATHS = [
+  'applicant-information-summary',
+  'applicant-contact-information',
+  'applicant-communication-preference',
+  'service-status',
+  'service-history',
+  'service-decoration',
+  'existing-loan-screener',
+  'loan-intent',
+  'loan-history',
+  'document-upload-summary',
+  'upload-supporting-documents',
+  'review-and-submit',
+  'confirmation',
+];
+
+function App({ location, children, loggedIn, router }) {
+  const formPath = window.location.pathname.split('/').pop();
+
+  if (!loggedIn && FORM_PATHS.includes(formPath)) {
+    router.push('/introduction');
+  }
```

Comment:
I'm not sure if this would work as expected. There are other paths like `/form-saved` & `/error` that I can think of... there might be more

Reply:
Tested with the two that you've mentioned, and those seem to already redirect back to introduction 🤷‍♂️